stdlib/universe/moving_average.go

```diff
@@ -124,6 +124,11 @@ type movingAverageTransformation struct {
 
 	n       int64
 	columns []string
+
+	i      []int
+	sum    []interface{}
```

Comment:
Can you comment on why you are using global slices?

Reply:
Since tables may be split into chunks, we want to be able to pick up where we left off when moving to a new table.